src/Build/Construction/Solution/SolutionConfigurationInSolution.cs

```diff
@@ -16,7 +16,9 @@ public sealed class SolutionConfigurationInSolution
         /// </summary>
         internal const char ConfigurationPlatformSeparator = '|';
```

Comment:
Used anymore?

Reply:
It's still used in `SolutionFile.TryParseConfigurationPlatform`, though now it looks like feature envy. I don't know what the "house rules" are in this repo, but I'd just inline the constant in `TryParseConfigurationPlatform` and remove this one personally.